testing/describes.js

```diff
@@ -304,53 +306,41 @@ function describeEnv(factory) {
    * @param {function(string, function())} describeFunc
    */
   const templateFunc = function (name, spec, fn, describeFunc) {
-    const fixtures = [new SandboxFixture(spec)];
-    factory(spec).forEach((fixture) => {
-      if (fixture && fixture.isOn()) {
-        fixtures.push(fixture);
-      }
-    });
-    return describeFunc(name, function () {
+    const fixtures = [new SandboxFixture(spec)].concat(
+      factory(spec)
+        .filter(Boolean)
+        .filter((fixture) => fixture.isOn())
```

Comment:
Nit: Why break this into two?

Reply:
Not sure actually. I think I started with switching to `factory(spec).filter(Boolean).forEach(...)`, and then when I switched from pushing into the array to just assigning the filtered array, left it split. I'll combine these.

Reply:
Done